Resources/Prototypes/_CD/Reagents/narcotics.yml

```
@@ -0,0 +1,102 @@
+- type: reagent
+  id: Dynamoxine
+  name: reagent-name-dynamoxine
+  group: Narcotics
+  desc: reagent-desc-dynamoxine
+  physicalDesc: reagent-physical-desc-sour
+  flavor: sharp
+  color: "#965A7D"
+  boilingPoint: 212.0
+  meltingPoint: 170.0
+  metabolisms:
+    Narcotic:
+      metabolismRate: 0.5
+      effects:
+      - !type:MovespeedModifier
+        walkSpeedModifier: 1.15
+        sprintSpeedModifier: 1.15
+      - !type:HealthChange
+        conditions:
+          - !type:ReagentThreshold
+            min: 10 # Careful.
+        damage:
+          types:
+            Bloodloss: 5
+            Asphyxiation: 5
+            Poison: 2
+      - !type:HealthChange
+        conditions:
+          - !type:ReagentThreshold
+          reagent: Epinephrine # DO NOT USE A FUCKING EPEN
+          min: 1
```

You are missing an indentation here, it is reading this as part of the conditions, not as part of the reagent threshold
```suggestion
            reagent: Epinephrine # DO NOT USE A FUCKING EPEN
            min: 1
```